lab-steven/server.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const morgan = require('morgan');
+const app = express();
+const cors = require('./lib/cors-middleware.js');
+const error = require('./lib/error-middleware.js');
+const studentRouter = require('./route/student-router.js');
+const PORT = process.env.PORT || 8080;
+
+app.use(morgan('dev'));
+app.use(studentRouter);
+app.use(cors);
```

Comment:
Your cors middleware should be before your api route, otherwise the headers won't be applied before the response is sent.